src/scythe-bidder/constants.ts

```diff
@@ -18,6 +18,42 @@ export const factions = [
   "Rusviet",
 ] as const;
 
+export const MATS_BASE = [
+  "Industrial",
+  "Engineering",
+  "Patriotic",
+  "Mechanical",
+  "Agricultural",
+] as const;
+
+export const FACTIONS_BASE = [
+  "Crimea",
+  "Saxony",
+  "Polania",
+  "Nordic",
+  "Rusviet",
+] as const;
+
+export const MATS_IFA = [
+  "Industrial",
+  "Engineering",
+  "Militant",
+  "Patriotic",
+  "Innovative",
+  "Mechanical",
+  "Agricultural",
+] as const;
+
+export const FACTIONS_IFA = [
+  "Togawa",
+  "Crimea",
+  "Saxony",
+  "Polania",
+  "Albion",
+  "Nordic",
+  "Rusviet",
+] as const;
+
```

Comment:
I should have noticed this before, but let's get rid of `mats` and `factions` and fix the imports in places that use them such as `game.ts`.